New Shuttle: Cybersun Adept

assignedgod opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Added a new expedition shuttle, the cybersun adept
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
This ship is one I've been thinking about for a while now, and has a lot of thought put into it. I wanted a clear central walkway, a prison (Connected to the medbay as you'll need to treat a prisoner more often than not), a nice salvage bay that was open yet compact, I wanted a storage room to hold equipment and a compact break room. The break room and storage room were made a bit more compact than I'd have liked but I didn't want o interrupt the walkway to the thruster bays too much. I went for a red and black theme without dipping too far into syndi territory and made sure to utilize decals to space the rooms and frame them.
## Technical details
<!-- Summary of code changes for easier review. -->
Adds a shipyard file and a shuttle
## How to test
<!-- Describe a procedure to test this feature, along with expected output/behavior. -->
Spawn in, buy the ship, ponder the awe 
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="942" height="750" alt="image" src="https://github.com/user-attachments/assets/9f3ff4ad-2c79-4ab1-a8b9-538fd04b19d8" />
<img width="952" height="741" alt="image" src="https://github.com/user-attachments/assets/53e360c0-0499-45f0-8ef8-f367e79a762f" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I have reviewed the [Ship Submission Guidelines](https://frontierstation.wiki.gg/wiki/Ship_Submission_Guidelines) if relevant.
- [X] I confirm that the content in this PR is my own work, and/or is properly attributed to the original author(s).
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->
 N/A
**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Added the Cybersun Adept, a new expedition/salvage ship!

assignedgod commented:
Forgot a gravity generator, fixing

assignedgod commented:
<img width="393" height="260" alt="image" src="https://github.com/user-attachments/assets/6bf48fc7-f8d2-4a95-8840-ba62d10a2d7b" />
Moved the safe around to accomodate

Mace70903 commented:
Hmm, I haven't loaded this in or anything but my thoughts just looking at the pictures.

- Toolboxes in engineering might be warranted, spare tools are always good.
- A material locker might be nice, just for a little bit of starting material for the lathes.
- The salvage bay is not nearly as open as I think you may have hoped. Think about the closest you could open the bay up to something, it's quite far, as the salvage bay is tucked very far into the ship and obstructed by the thruster nacelles. Swapping engineering/med with the salvage bay might make it much easier to park those blast doors next to something, though I know from experience doing that much modification would be a chore.
- I don't imagine you'll need more that one tank dispenser.
- A lot of the outer walls are diagonals, don't know what your mapping standards are like, I imagine looser, I must say it does look a little jarring though. I think it might look nicer with diagonal walls joining up a lot of those spots so there aren't floating wall pieces.
- The bridge could do with expanding one tile up and fitting things like the fax machine in it I think, it looks a little cramped. Especially for a hypothetical two people.
- The tile work is a little same-y. Maybe swap it up from dark steel tiles a little more

assignedgod commented:
I'll definitely take that into account, thanks for the feedback! :3

Zenixic commented:
this does look awkward
<img width="878" height="175" alt="image" src="https://github.com/user-attachments/assets/083aabcd-c837-4a46-b38a-0983f733623b" />

this would look less awkward
<img width="988" height="225" alt="image" src="https://github.com/user-attachments/assets/f3b0e5fd-873c-41b3-867b-b8d9b6a562fe" />

Missing a pipe here
<img width="398" height="264" alt="image" src="https://github.com/user-attachments/assets/4d861a38-91d3-4c97-8236-b6ac03e67255" />

Cannot interact with the gun rack from northeast side
<img width="131" height="171" alt="image" src="https://github.com/user-attachments/assets/fe6dd1c5-3224-450a-8252-a3b3b08084ee" />
but is from the west side, maybe adjust it to face north?
<img width="140" height="168" alt="image" src="https://github.com/user-attachments/assets/2a94ceef-b475-4550-ab91-46583b541016" />

I dont think frontier adjusted this fill? but this medical wall locker starts empty
<img width="580" height="190" alt="image" src="https://github.com/user-attachments/assets/ab2edbeb-af46-4c42-92d2-9c617cf3aec9" />

Cygnall commented:
Bridges typically use the Blue Command doors

assignedgod commented:
<img width="1021" height="742" alt="image" src="https://github.com/user-attachments/assets/d92d3fe9-e3ec-48ee-93db-0e2fc1f0bfbf" />
<img width="1079" height="834" alt="image" src="https://github.com/user-attachments/assets/a9e32b6e-73ab-4043-a9e3-95f8fd116796" />
Thanks all for the excellent feedback! I opted not to move the salvage bay doors, if it bothers someone the ship is modifiable enough that the front railings on the wings can be clipped, or an rcd and slight thruster adjustment will also work. I opted to replace the weapon rack with a materials locker. I have changed up the tiling in the central hub and salvage bay to improve the look of the ship. I added a fax machine and water cooler to the cockpit and lengthened it to make it feel less cramped.

Zenixic commented:
There is still this bit that looks like a disconnected wall (dont forget to add to the other side to make it symmetrical)
<img width="217" height="180" alt="image" src="https://github.com/user-attachments/assets/b96967c0-dc7a-4d4e-8090-9b3ffb9fa4f7" />

Dont forget to paint your new pipes!
<img width="171" height="88" alt="image" src="https://github.com/user-attachments/assets/6bf1d405-9661-4262-b6a6-cd535513114a" />

assignedgod commented:
<img width="189" height="150" alt="image" src="https://github.com/user-attachments/assets/c9aa1fcf-93be-494c-9fe9-2797b6d1bc58" />
<img width="1184" height="215" alt="image" src="https://github.com/user-attachments/assets/2e2364f9-0951-43ba-9888-65b07866bf6e" />
<img width="540" height="204" alt="image" src="https://github.com/user-attachments/assets/c1fb15c5-f850-464f-ab67-55a3c9715132" />
Solid advice, thanks, integrated em all :3
